### ISSUE TITLE
Low correlation coefficient of size model

### ISSUE THREAD

**Issue by charan03012000** [MouseLand/cellpose#455]
Hi, I tried replicating the results using the following command specified in the documentation(from scratch training of cyto model).
`python -m cellpose --train --train_size --use_gpu --dir ~/cellpose_dataset/train/ --test_dir ~/cellpose_dataset/test/ --img_filter _img --pretrained_model None --chan 2 --chan2 1`

And here is the whole training output.

creating new log file
WRITING LOG OUTPUT TO /root/.cellpose/run.log
**TORCH CUDA version installed and working.**
>>>> using GPU
Not all flows are present. Run flow generation again.
Not all flows are present. Run flow generation again.
>>>> training from scratch
>>>> during training rescaling images to fixed diameter of 30.0 pixels
Training with rescale = 1.00
/kaggle/working/cellpose/transforms.py:365: UserWarning: 'chan2 (opt)' has value range of ZERO, can instead set chan2 to 0
  warnings.warn("'chan2 (opt)' has value range of ZERO, can instead set chan2 to 0")
computing flows for labels
100%|█████████████████████████████████████████| 540/540 [01:03<00:00,  8.51it/s]
computing flows for labels
100%|███████████████████████████████████████████| 68/68 [00:07<00:00,  8.82it/s]
1 train images with number of masks less than min_train_masks (5), removing from train set
>>>> median diameter set to = 30
>>>> training network with 2 channel input <<<<
>>>> LR: 0.20000, batch_size: 8, weight_decay: 0.00001
>>>> ntrain = 539, ntest = 68
>>>> nimg_per_epoch = 539
Epoch 0, Time 21.3s, Loss 3.7491, Loss Test 3.8896, LR 0.0000
saving network parameters to /kaggle/working/cellpose_dataset/train/models/cellpose_residual_on_style_on_concatenation_off_train_2022_02_23_17_59_29.478325
Epoch 5, Time 92.1s, Loss 2.1066, Loss Test 1.8874, LR 0.1111
Epoch 10, Time 162.9s, Loss 1.6360, Loss Test 1.7682, LR 0.2000
Epoch 20, Time 303.5s, Loss 1.4336, Loss Test 1.4661, LR 0.2000
Epoch 30, Time 444.6s, Loss 1.2885, Loss Test 1.3746, LR 0.2000
Epoch 40, Time 585.4s, Loss 1.2307, Loss Test 1.3586, LR 0.2000
Epoch 50, Time 725.8s, Loss 1.1837, Loss Test 1.3641, LR 0.2000
Epoch 60, Time 866.2s, Loss 1.1475, Loss Test 1.2155, LR 0.2000
Epoch 70, Time 1006.5s, Loss 1.1143, Loss Test 1.2150, LR 0.2000
Epoch 80, Time 1147.1s, Loss 1.0879, Loss Test 1.1627, LR 0.2000
Epoch 90, Time 1287.3s, Loss 1.0681, Loss Test 1.1849, LR 0.2000
Epoch 100, Time 1427.8s, Loss 1.0518, Loss Test 1.1871, LR 0.2000
saving network parameters to /kaggle/working/cellpose_dataset/train/models/cellpose_residual_on_style_on_concatenation_off_train_2022_02_23_17_59_29.478325
Epoch 110, Time 1568.4s, Loss 1.0496, Loss Test 1.1464, LR 0.2000
Epoch 120, Time 1709.0s, Loss 1.0235, Loss Test 1.1944, LR 0.2000
Epoch 130, Time 1849.1s, Loss 1.0101, Loss Test 1.1312, LR 0.2000
Epoch 140, Time 1989.5s, Loss 1.0050, Loss Test 1.1961, LR 0.2000
Epoch 150, Time 2129.7s, Loss 1.0055, Loss Test 1.1186, LR 0.2000
Epoch 160, Time 2270.0s, Loss 0.9842, Loss Test 1.0636, LR 0.2000
Epoch 170, Time 2410.2s, Loss 0.9689, Loss Test 1.1212, LR 0.2000
Epoch 180, Time 2550.6s, Loss 0.9735, Loss Test 1.0920, LR 0.2000
Epoch 190, Time 2690.7s, Loss 0.9561, Loss Test 1.0937, LR 0.2000
Epoch 200, Time 2831.3s, Loss 0.9576, Loss Test 1.0664, LR 0.2000
saving network parameters to /kaggle/working/cellpose_dataset/train/models/cellpose_residual_on_style_on_concatenation_off_train_2022_02_23_17_59_29.478325
Epoch 210, Time 2969.6s, Loss 0.9442, Loss Test 1.0544, LR 0.2000
Epoch 220, Time 3108.0s, Loss 0.9310, Loss Test 1.0849, LR 0.2000
Epoch 230, Time 3246.8s, Loss 0.9286, Loss Test 1.0250, LR 0.2000
Epoch 240, Time 3386.1s, Loss 0.9216, Loss Test 1.1280, LR 0.2000
Epoch 250, Time 3525.5s, Loss 0.9141, Loss Test 1.0625, LR 0.2000
Epoch 260, Time 3664.6s, Loss 0.9182, Loss Test 1.0686, LR 0.2000
Epoch 270, Time 3803.7s, Loss 0.9177, Loss Test 1.0603, LR 0.2000
Epoch 280, Time 3942.8s, Loss 0.8947, Loss Test 1.0736, LR 0.2000
Epoch 290, Time 4082.2s, Loss 0.8901, Loss Test 1.0427, LR 0.2000
Epoch 300, Time 4221.7s, Loss 0.9010, Loss Test 1.0627, LR 0.2000
saving network parameters to /kaggle/working/cellpose_dataset/train/models/cellpose_residual_on_style_on_concatenation_off_train_2022_02_23_17_59_29.478325
Epoch 310, Time 4360.9s, Loss 0.8896, Loss Test 1.0497, LR 0.2000
Epoch 320, Time 4500.3s, Loss 0.8911, Loss Test 1.0199, LR 0.2000
Epoch 330, Time 4639.9s, Loss 0.9349, Loss Test 1.1103, LR 0.2000
Epoch 340, Time 4779.6s, Loss 0.8878, Loss Test 1.0438, LR 0.2000
Epoch 350, Time 4919.0s, Loss 0.8890, Loss Test 1.0550, LR 0.2000
Epoch 360, Time 5058.3s, Loss 0.9145, Loss Test 1.0095, LR 0.2000
Epoch 370, Time 5197.2s, Loss 0.8821, Loss Test 1.1714, LR 0.2000
Epoch 380, Time 5336.3s, Loss 0.8710, Loss Test 1.0266, LR 0.2000
Epoch 390, Time 5475.3s, Loss 0.8714, Loss Test 1.0399, LR 0.2000
Epoch 400, Time 5614.0s, Loss 0.8601, Loss Test 1.1161, LR 0.2000
saving network parameters to /kaggle/working/cellpose_dataset/train/models/cellpose_residual_on_style_on_concatenation_off_train_2022_02_23_17_59_29.478325
Epoch 410, Time 5752.8s, Loss 0.8569, Loss Test 1.0136, LR 0.1000
Epoch 420, Time 5891.4s, Loss 0.8210, Loss Test 0.9841, LR 0.0500
Epoch 430, Time 6030.0s, Loss 0.7970, Loss Test 0.9811, LR 0.0250
Epoch 440, Time 6168.6s, Loss 0.7893, Loss Test 0.9856, LR 0.0125
Epoch 450, Time 6307.0s, Loss 0.7821, Loss Test 0.9730, LR 0.0063
Epoch 460, Time 6444.9s, Loss 0.7764, Loss Test 0.9786, LR 0.0031
Epoch 470, Time 6583.1s, Loss 0.7767, Loss Test 0.9808, LR 0.0016
Epoch 480, Time 6721.5s, Loss 0.7758, Loss Test 0.9805, LR 0.0008
Epoch 490, Time 6859.7s, Loss 0.7756, Loss Test 0.9734, LR 0.0004
saving network parameters to /kaggle/working/cellpose_dataset/train/models/cellpose_residual_on_style_on_concatenation_off_train_2022_02_23_17_59_29.478325
>>>> model trained and saved to /kaggle/working/cellpose_dataset/train/models/cellpose_residual_on_style_on_concatenation_off_train_2022_02_23_17_59_29.478325
/kaggle/working/cellpose/transforms.py:363: UserWarning: chan to seg' has value range of ZERO
  warnings.warn("chan to seg' has value range of ZERO")
ran 2 epochs in 36.882 sec
ran 4 epochs in 73.480 sec
ran 6 epochs in 110.207 sec
ran 8 epochs in 146.850 sec
ran 10 epochs in 183.583 sec
train correlation: 0.7740
test correlation: 0.4798
model saved to /kaggle/working/cellpose_dataset/train/models/cellpose_residual_on_style_on_concatenation_off_train_2022_02_23_17_59_29.478325_size.npy
0%|          | 0/68 [00:00<?, ?it/s]
1%|1         | 1/68 [00:00<00:30,  2.20it/s]
3%|2         | 2/68 [00:00<00:29,  2.27it/s]
4%|4         | 3/68 [00:01<00:31,  2.09it/s]
6%|5         | 4/68 [00:01<00:27,  2.35it/s]
7%|7         | 5/68 [00:02<00:26,  2.34it/s]
9%|8         | 6/68 [00:02<00:24,  2.58it/s]
10%|#         | 7/68 [00:02<00:24,  2.47it/s]
12%|#1        | 8/68 [00:03<00:32,  1.85it/s]
13%|#3        | 9/68 [00:04<00:31,  1.85it/s]
15%|#4        | 10/68 [00:04<00:28,  2.04it/s]
16%|#6        | 11/68 [00:05<00:28,  2.03it/s]
18%|#7        | 12/68 [00:05<00:24,  2.25it/s]
19%|#9        | 13/68 [00:06<00:26,  2.11it/s]
21%|##        | 14/68 [00:06<00:27,  1.96it/s]
22%|##2       | 15/68 [00:09<01:00,  1.14s/it]
24%|##3       | 16/68 [00:09<00:49,  1.05it/s]
25%|##5       | 17/68 [00:10<00:42,  1.19it/s]
26%|##6       | 18/68 [00:10<00:38,  1.31it/s]
28%|##7       | 19/68 [00:11<00:31,  1.54it/s]
29%|##9       | 20/68 [00:11<00:29,  1.61it/s]
31%|###       | 21/68 [00:12<00:33,  1.39it/s]
32%|###2      | 22/68 [00:13<00:35,  1.28it/s]
34%|###3      | 23/68 [00:14<00:31,  1.41it/s]
35%|###5      | 24/68 [00:14<00:30,  1.46it/s]
37%|###6      | 25/68 [00:16<00:35,  1.21it/s]
38%|###8      | 26/68 [00:16<00:30,  1.36it/s]
40%|###9      | 27/68 [00:16<00:23,  1.72it/s]
41%|####1     | 28/68 [00:17<00:29,  1.34it/s]
43%|####2     | 29/68 [00:18<00:24,  1.59it/s]
44%|####4     | 30/68 [00:18<00:21,  1.78it/s]
46%|####5     | 31/68 [00:19<00:21,  1.68it/s]
47%|####7     | 32/68 [00:19<00:19,  1.82it/s]
49%|####8     | 33/68 [00:20<00:17,  1.97it/s]
50%|#####     | 34/68 [00:21<00:20,  1.62it/s]
51%|#####1    | 35/68 [00:21<00:21,  1.55it/s]
53%|#####2    | 36/68 [00:22<00:19,  1.64it/s]
54%|#####4    | 37/68 [00:22<00:17,  1.80it/s]
56%|#####5    | 38/68 [00:23<00:16,  1.80it/s]
57%|#####7    | 39/68 [00:23<00:16,  1.72it/s]
59%|#####8    | 40/68 [00:24<00:15,  1.85it/s]
60%|######    | 41/68 [00:24<00:14,  1.82it/s]
62%|######1   | 42/68 [00:26<00:18,  1.37it/s]
63%|######3   | 43/68 [00:26<00:18,  1.33it/s]
65%|######4   | 44/68 [00:27<00:16,  1.41it/s]
66%|######6   | 45/68 [00:30<00:32,  1.41s/it]
68%|######7   | 46/68 [00:33<00:39,  1.80s/it]
69%|######9   | 47/68 [00:34<00:31,  1.49s/it]
71%|#######   | 48/68 [00:34<00:22,  1.15s/it]
72%|#######2  | 49/68 [00:35<00:23,  1.23s/it]
74%|#######3  | 50/68 [00:36<00:18,  1.05s/it]
75%|#######5  | 51/68 [00:37<00:18,  1.07s/it]
76%|#######6  | 52/68 [00:38<00:14,  1.09it/s]
78%|#######7  | 53/68 [00:38<00:11,  1.31it/s]
79%|#######9  | 54/68 [00:40<00:14,  1.01s/it]
81%|########  | 55/68 [00:40<00:12,  1.05it/s]
82%|########2 | 56/68 [00:41<00:09,  1.22it/s]
84%|########3 | 57/68 [00:42<00:08,  1.22it/s]
85%|########5 | 58/68 [00:44<00:11,  1.17s/it]
87%|########6 | 59/68 [00:45<00:11,  1.30s/it]
88%|########8 | 60/68 [00:46<00:08,  1.11s/it]
90%|########9 | 61/68 [00:47<00:06,  1.04it/s]
91%|#########1| 62/68 [00:48<00:07,  1.22s/it]
93%|#########2| 63/68 [00:49<00:05,  1.05s/it]
94%|#########4| 64/68 [00:50<00:03,  1.15it/s]
96%|#########5| 65/68 [00:50<00:02,  1.32it/s]
97%|#########7| 66/68 [00:51<00:01,  1.08it/s]
99%|#########8| 67/68 [00:52<00:00,  1.28it/s]
100%|##########| 68/68 [00:52<00:00,  1.52it/s]
100%|##########| 68/68 [00:52<00:00,  1.29it/s]
style test correlation: 0.5219; final test correlation: 0.7290

Im not able to figure why would the correlation be not around 0.9 on the test set
